dask_ml/model_selection/_incremental.py

```diff
@@ -154,6 +169,10 @@ def _fit(
     y_train = sorted(futures_of(y_train), key=lambda f: f.key)
     assert len(X_train) == len(y_train)
 
+    train_eg = yield client.map(len, y_train)
```

Comment:
I don't understand this map call. `y_train` is a Dask array right?

Reply:
`y_train` is a list of futures to ndarrays. See the reassignment with `future_of` three lines above this.

``` 
>>> type(client.compute(y_train))
list
>>> [type(x) for x in client.compute(y_train)]
[<class 'numpy.ndarray'>, <class 'numpy.ndarray'>]
>>> [x.shape for x in client.compute(y_train)]
[(20,), (20,)]
```